napari/layers/utils/layer_utils.py

```diff
@@ -417,69 +338,6 @@ def coerce_current_properties(
     return coerced_current_properties
 
 
-def dataframe_to_properties(
```

Comment:
Related to the public/private utils discussion, this and other functions here are technically public, but I'm just deleting them. Is that OK?

Reply:
These are back, so no problem anymore.